tests/dashbio_demos/app_circos.py

```diff
@@ -1130,8 +1128,8 @@ def layout():
                                 'marginTop': '5px',
                                 'marginBottom': '10px',
                             },
-                            n_fixed_rows=1,
-                            n_fixed_columns=1
+                            fixed_rows=1,
+                            fixed_columns=1
```

Comment:
closer... in addition to the name change, these props are now objects `{headers: bool, data: int}`